components/inputs/test/input-time.visual-diff.js

```diff
@@ -0,0 +1,37 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+
+describe('d2l-input-time', () => {
+
+	const visualDiff = new VisualDiff('input-time', __dirname);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await browser.newPage();
+		await page.setViewport({width: 800, height: 800, deviceScaleFactor: 2});
+		await page.goto(`${visualDiff.getBaseUrl()}/components/inputs/test/input-time.visual-diff.html`, {waitUntil: ['networkidle0', 'load']});
+		await page.bringToFront();
+	});
+
+	after(() => browser.close());
+
+	[
+		'basic',
+		'disabled',
+		'labelled',
+		'label-hidden',
+	].forEach((name) => {
+		it(name, async function() {
+			const rect = await visualDiff.getRect(page, `#${name}`);
+			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+		});
+		it(`${name}-focus`, async function() {
```

Comment:
I know text inputs likely did this, but because time input just uses text input, I feel like just having a single focus visual test is likely enough -- not one for each of the different states.